experimenter/experimenter/nimbus_ui_new/urls.py

```diff
@@ -52,4 +54,10 @@
         NimbusExperimentsCreateView.as_view(),
         name="nimbus-new-create",
     ),
+    re_path(r"^(?P<slug>[\w-]+)/subscribe/", SubscribeView.as_view(), name="subscribe"),
+    re_path(
+        r"^(?P<slug>[\w-]+)/unsubscribe/",
+        UnsubscribeView.as_view(),
+        name="unsubscribe",
+    ),
```

Comment:
Oh these url names should be

```
nimbus-new-subscribe
nimbus-new-unsubscribe
```

so we dont' accidentally get collissions with other views 🙏